packages/CLAPACK/Makefile

```diff
@@ -8,16 +8,16 @@ ROOT=$(abspath .)
 SRC=$(ROOT)/CLAPACK-WA
 
 
-all: $(SRC)/lapack_WA.bc
+all: $(SRC)/lapack_WA.a
```

Comment:
I think we are already building .a files here; this is a typo in the Makefile. It is good to correct it either way.

Reply:
yeah it is